python/ct/crypto/tools/verify_util.py

```diff
@@ -61,15 +61,20 @@ def main(argv):
     except gflags.FlagsError as e:
         exit_with_message("Error parsing flags: %s" % e)
 
-    command, cert_file = (argv[0:2])
+    command = argv[0]
+    chain = argv[1:]
 
     if command != "verify_sct":
         exit_with_message("Unknown command %s" % command)
 
-    if not cert_file:
+    if not chain:
         exit_with_message("No certificate file given")
 
-    verify_sct(cert.Certificate.from_pem_file(cert_file, strict_der=False),
+    chain = map(lambda name:
+                    cert.Certificate.from_pem_file(name, strict_der = True),
```

Comment:
The Certificate.from_pem_file returns an iterable of certs from the file, which is why you don't need to specify multiple files.